.storybook/webpack.config.js

```diff
@@ -17,5 +17,12 @@ module.exports = {
         );
       },
     ),
+    new webpack.DefinePlugin({
+      'process.env': {
+        SIMORGH_EMBED_BASE_URL: JSON.stringify(
+          process.env.SIMORGH_EMBED_BASE_URL,
+        ),
+      },
+    }),
```

Comment:
Why is this needed/what is it doing?

Reply:
Storybook does not have access to `process.env` variables, this adds the embed base URL to storybook.